src/components/header.tsx

```diff
@@ -34,33 +42,51 @@ const useStyles = makeStyles(() =>
 const Header: React.FC<HeaderProps> = ({ siteTitle = '' }: HeaderProps) => {
   const classes = useStyles();
 
-  const [open, setOpen] = React.useState(false);
+  const [signinDialogOpen, setSigninDialogOpen] = React.useState(false);
+  const [navbarOpen, setNavbarOpen] = React.useState(false);
+
+  const handleClickOpenSigninDialog = (): void => {
```

Comment:
this handler can just be named `handleOpenSigninDialog`

Reply:
completed!